ragweed/framework.py

```diff
@@ -209,14 +207,9 @@ def __init__(self, config):
 
     def get(self, storage_class):
         assert(self.storage_classes != None)
-        try:
-            if not storage_class:
-                storage_class = 'STANDARD'
-            sc = self.storage_classes[storage_class]
-        except:
-            eq('could not find storage class ' + storage_class, 0)
-
-        return sc
+        if not storage_class:
+            storage_class = 'STANDARD'
+        return self.storage_classes[storage_class] # may raise KeyError
```

Comment:
@cbodley Was your logic behind this case that a `KeyError` would be thrown if the `storage_class` didn't exist in `storage_classes` and that would be adequate to diagnose what was wrong if this `KeyError` was hit?

Reply:
that's right